### ISSUE TITLE
ER: issues on  CONTRIBUTING PAGE

### ISSUE THREAD

**Issue by garrettallen0** [hackforla/website#4559]
### Emergent Requirement - Problem 

3 separate problems on CONTRIBUTING pages I list below. 

1) Git GUI

On [this page](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#27-working-on-an-issue), on 2.7, the initial bullet point under the heading is incorrectly links to a "Create new page." 

It is supposed to link to [this site](https://github.com/hackforla/website/wiki/How-to-Use-Git-GUI-(Graphical-user-Interface)-in-Visual-Studios-Code). Other versions of the CONTRIBUTING page do this correctly, but this version does not. 

2) Handling SSH 

On [this page](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27f-working-on-an-issue-6-handling-ssh-authorization-errors), on 2.7.f, step 4 reads like this: 

> Next you need to add your ssh key to your ssh agent.
> 
> `ssh-add -K ~/.ssh/id_ed25519`
> 

But it doesn't say you need to use _you own_ ssh agent, not "id_ed25519". This was a stumbling block to me as a new dev. 

3) Duplication of delete instructions 

On [this page](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-3-pull-requests), under "3.1.b Complete pull request on Hack for LA website repo", the following reads like a duplication to me. 

> Once your pull request is merged you can delete your branch with the following command:
> 
>` git branch -d update-give-link-2093`
> 
> Now you can move on to your next issue and create a new branch. (This ensures you don’t accidentally include the changes from your previous branch in your new branch)
> 
> From here, once your pull request is approved and merged you can pull the recent merge from the Hack For LA repository and delete your local branch:
> 
> `git pull upstream gh-pages`
> `git branch -d update-give-link-2093`

### Issue you discovered this emergent requirement in
- my first issue, #4394 

### Date discovered:
4/18/23

### Did you have to do something temporarily
- [x] NO

### Who was involved
@garrettallen0 

### What happens if this is not addressed

Confusion to new developers. 

### Recommended Action Items
- [ ] Make a new issue
- [ ] Discuss with team
- [ ] Let a Team Lead know

**Comment by garrettallen0** [comment]
Hi @jdingeman, could you give me feedback on this ER? Is it written intelligibly? Do I have the necessary labels? Anything I'm missing?

**Comment by Hritik1503** [comment]
I am interested to work on this issue under Quine. So please assign me this task to me.

**Comment by ExperimentsInHonesty** [comment]
@garrettallen0 The ER is good, but not sure why it doesnt have the same formatting as other ERs.  Did you copy from an existing ER instead of using the [ER tempate](https://github.com/hackforla/website/issues/new?assignees=&labels=size%3A+0.25pt&template=emergent-request.md&title=ER%3A+%5Breplace+with+info+%5D+)?  

Please fix the formatting.  After that, I believe this ER is ready for 3 issues to get made from it.  I have added the label `ready for dev lead` to it.  But you are welcome to draft the 3 issues action items in the Potential solutions section.

**Comment by garrettallen0** [comment]
Thanks @ExperimentsInHonesty. No, I deleted the "###" the first time around simply because I didn't know I needed them! I think it is formatted correctly now.

**Comment by JessicaLucindaCheng** [comment]
> 1. Git GUI
> On [this page](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#27-working-on-an-issue), on 2.7, the initial bullet point under the heading is incorrectly links to a "Create new page."
> 
> It is supposed to link to [this site](https://github.com/hackforla/website/wiki/How-to-Use-Git-GUI-(Graphical-user-Interface)-in-Visual-Studios-Code). Other versions of the CONTRIBUTING page do this correctly, but this version does not.

The link (https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#27-working-on-an-issue) is to a particular commit of the CONTRIBUTING.md file. If it works on the current file (https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md), then the problem has already been fixed, so an issue doesn't need to be written for that part of the ER.

**Comment by roslynwythe** [comment]
- Problem 1 Git GUI - has been resolved in the current version 
- Problem 2 SSH 
   - I agree that this can be clarified.  https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md #5693 (Mac) and #5694 (WIndows).  
   -  I suggest that if the ~/.ssh directory does not exist, it should be created prior to running `ssh-keygen`.  #5695 and #and #6240
- Problem 3 Duplication of delete instructions - #5696 

I would like to add:
- Problem 4  I suggest removing the statements regarding "pushing local commits to your repository" from Section 2.7.a #5697
- Problem 5 - 2.7.b I suggest adding language to ensure that devs have checked out `gh-pages` before proceeding to create a new issue branch #5698
